language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -325,3 +385,84 @@ callProcessSilent cmd args = do
       hPutStrLn stderr $ unlines ["stdout:", out]
       hPutStrLn stderr $ unlines ["stderr: ", err]
       exitFailure
+
+writeTsConfig :: FilePath -> IO ()
+writeTsConfig dir = writeFileUTF8 (dir </> "tsconfig.json") $ unlines
+    [ "{"
+    , "  \"compilerOptions\": {"
+    , "    \"target\": \"es5\","
+    , "    \"lib\": ["
+    , "      \"dom\","
+    , "      \"es2015\""
+    , "     ],"
+    , "    \"strict\": true,"
+    , "    \"noUnusedLocals\": true,"
+    , "    \"noUnusedParameters\": false,"
+    , "    \"noImplicitReturns\": true,"
+    , "    \"noFallthroughCasesInSwitch\": true,"
+    , "    \"outDir\": \"lib\","
+    , "    \"module\": \"commonjs\","
+    , "    \"declaration\": true,"
+    , "    \"sourceMap\": true"
+    , "    },"
+    , "  \"include\": [\"src/**/*.ts\"],"
+    , "}"
+    ]
+
+writePackageJson :: FilePath -> String -> IO ()
+writePackageJson dir name = writeFileUTF8 (dir </> "package.json") $ unlines
+        ["{"
+        , "  \"private\": true,"
+        , "  \"name\": \"" <> name <> "\","
+        , "  \"version\": \"0.13.53\","
+        , "  \"description\": \"Produced by daml2ts\","
+        , "  \"license\": \"Apache-2.0\","
+        , "  \"dependencies\": {"
+        , "    \"@daml/types\": \"0.13.53\","
```

Comment:
That should be the current SDK version too.

Reply:
Again, well spotted.

Reply:
Actually, we might need `0.0.0-SNAPSHOT` here in order to use `@daml/types` from `HEAD`.  The `build-and-list` test should demonstrate how to depend on these.